Apps/Common/src/Services/AccessTokenService.cs

```diff
@@ -72,25 +72,26 @@ public class AccessTokenService : IAccessTokenService
         public async Task<RequestResult<TokenSwapResponse>> GetPhsaAccessToken()
         {
             using Activity? activity = Source.StartActivity();
-            string? hdid = this.authenticationDelegate.FetchAuthenticatedUserHdid();
+            string? identifier = this.authenticationDelegate.FetchAuthenticatedUserIdentifier();
```

Comment:
nit: userid is probably better than identifier

Reply:
Changed.